src/mainwindow.ui

```diff
@@ -117,10 +117,17 @@
     <addaction name="separator"/>
     <addaction name="action_anim_delete"/>
    </widget>
+   <widget class="QMenu" name="menuUtilities">
+    <property name="title">
+     <string>Utilities</string>
```

Comment:
Cool. I will work on my features tomorrow afternoon, and sunday. So monday should be a good cut. 
My main ideas for modern GL are currently on ice. 
I thought about using instances, turns out they are utterly broken with the linux catalyst driver. I will experiment a bit with Geometry Shaders, but the big bottleneck seems to be the python code. I've got some ideas for improved caching of geometry data that don't require modern openGL.
If I implement openGL 3.x code, I will leave the old rendering code as a fallback.